httpmw/operationid/operationid.go

```diff
@@ -0,0 +1,92 @@
+package operationid
```

Comment:
Better to rename this to be requestid to be consistent with the gRPC middleware.

Reply:
done